<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeServerPasswordDialog</class>
 <widget class="QDialog" name="ChangeServerPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Server Password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="oldPassLabel">
       <property name="text">
        <string>Current password:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="oldPassBox">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="newPassLabel">
       <property name="text">
        <string>New password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="newPassBox">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="confirmLabel">
       <property name="text">
        <string>Confirm password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="confirmBox">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="currentPasswordError">
     <property name="text">
      <string>The current password is incorrect.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #b00020;</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="newPasswordError">
     <property name="text">
      <string>The new passwords do not match.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #b00020;</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="noteRowWidget" native="true">
     <layout class="QHBoxLayout" name="noteRowLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="noteIconLabel">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noteLabel">
        <property name="text">
         <string>Passwords are case sensitive. If you lose or forget the password, it cannot be recovered.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="poolSecretRowWidget" native="true">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="poolSecretRowLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="poolSecretIconLabel">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="poolSecretLabel">
        <property name="text">
         <string>To reduce the risk of unauthorized access to your system, it is recommended that you also rotate the pool secret.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
